### PR TITLE
Add nGraph files for BERT training

### PR DESCRIPTION
**Pull request by baojun-nervana**
The PR includes the followings:
1. updated train.py to use executor
2. added train.sh to run bert training with ngraph. It will
    - download the model from LARK repo
    - copy and overwrite train.sh in BERT directory
    - set up env exports
    - run bert training with FLAGS_use_ngraph=true train.py --with_args
3. added readme for instructions

CC. @tensor-tang @luotao1  @mozga-intel @lzhao4ever

### CODE REVIEW

**Comment by baojun-nervana** [comment]
@tensor-tang @luotao1 Here is the script and file to run BERT training with nGraph

**Comment by Xreki** [comment]
Why not submit to https://github.com/PaddlePaddle/LARK?

**Comment by luotao1** [comment]
Discussed with @Xreki, benchmark repo could only have scripts like `run.sh` and `Readme.md` files, thus, could you update the train.py in LARK repo?

**Comment by baojun-nervana** [comment]
> Discussed with @Xreki, benchmark repo could only have scripts like `run.sh` and `Readme.md` files, thus, could you update the train.py in LARK repo?

train.py was renamed as train_ngraph.py and submitted PR to lark repo
https://github.com/PaddlePaddle/LARK/pull/158

File: NeuralMachineTranslation/BERT/fluid/train/ngraph/train.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -xe
+
+export OMP_NUM_THREADS=$(nproc)
+export KMP_AFFINITY=granularity=fine,compact,1,0
+
+git clone https://github.com/PaddlePaddle/LARK.git
```
Comment (tensor-tang) [inline NeuralMachineTranslation/BERT/fluid/train/ngraph/train.sh]: 明确说明下进入 指定目录就好，不要额外下载了。

留一个`BERT_ROOT=<your_path>`就好

File: NeuralMachineTranslation/BERT/fluid/train/ngraph/README_ngraph.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+To run Bert training:
+
+```
+./train.sh
+```
+
+The script will do the following:
+
+1. git clone paddlepaddle/LARK. The model is from https://github.com/PaddlePaddle/LARK/tree/develop/BERT
```
Comment (tensor-tang) [inline NeuralMachineTranslation/BERT/fluid/train/ngraph/README_ngraph.md]: 这里说明下载方式，`git clone https://github.com/PaddlePaddle/LARK.git`

**Pull request merged**